### PR TITLE
More docs

### PR DESCRIPTION
**Pull request by jdent02**

### CODE REVIEW

File: docs/panels/object/object_options.rst [hunk 1]
```diff
@@ -1,2 +1,20 @@
...
+==============
+
+.. image:: /_static/screenshots/object_panels/object_options.PNG
+
+|
+
+- SSS Set:
+	- This assigns the object to an SSS set.  The sets themselves are set in the :ref:`SSS Sets <label_sss_sets>` panel.
+- Nested Dielectric Medium Priority:
+	- This allows you to set the priority for this object where it intersects with another object.  A higher number equals a lower priority (i.e. 1 takes precedence over 2).  This allows you to correctly render intersecting media like fluid in a glass.
```
Comment (dictoon) [inline docs/panels/object/object_options.rst]: It's the opposite, high number have higher priority.

File: docs/panels/object/object_options.rst [hunk 2]
```diff
@@ -1,2 +1,20 @@
...
+- Ray Bias Method:
+	- This allows you to set how a ray is biased (offset from the actual shading point).
+- Ray Bias Distance:
+	- This sets how far away from the shading point the ray will be biased.
+	
+.. _label_obj_alpha:
+
+- Object Alpha:
+	- This sets the opacity of the object.  A texture may be used to create patterns.  The effects of this option are combined with any material based alpha mapping.
```
Comment (dictoon) [inline docs/panels/object/object_options.rst]: *material-based*

File: docs/panels/world/world_panel.rst [hunk 3]
```diff
@@ -1,5 +1,6 @@
 World Panel
 ===========
+The world panel contains settings that affect the entire 'world' of the 3D scene.
```
Comment (dictoon) [inline docs/panels/world/world_panel.rst]: Use double quotes: "world"

**Comment by jdent02** [comment]
Should be fixed.

**Pull request merged**